src/worker.py

```
@@ -1301,4 +1339,196 @@ async def on_fetch(request, env):
+    if existing:
+        if existing["status"] == "accepted":
+            return err("Already connected", 409)
+        if existing["status"] == "pending":
+            return err("Connection request already sent", 409)
+        if existing["status"] == "declined":
+            return err("Your previous request was declined", 409)
+    try:
+        body = await req.json()
+        if not isinstance(body, dict):
+            body = {}
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)